pkg/local_object_storage/metabase/shard_id.go

```diff
@@ -6,8 +6,9 @@ import (
 )
 
 var (
-	shardInfoBucket = []byte{shardInfoPrefix}
-	shardIDKey      = []byte("id")
+	shardInfoBucket     = []byte{shardInfoPrefix}
```

Comment:
> the error is probably elsewhere

PR fixes the migration that was mentioned in the issue (`v0.32.0` -> `v0.32.0-73-g01ddb3f8`), I checked that

> we should treat missing bucket as 0 version

some of our components create buckets before `meta.Init` so it could not be missing at all. changed the logic a little, recheck, please